src/libraries/System.Private.CoreLib/src/System/Reflection/RuntimeReflectionExtensions.cs

```diff
@@ -3,14 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 
 namespace System.Reflection
 {
     public static partial class RuntimeReflectionExtensions
     {
         private const BindingFlags Everything = BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static | BindingFlags.Instance;
 
-        public static IEnumerable<FieldInfo> GetRuntimeFields(this Type type)
+        public static IEnumerable<FieldInfo> GetRuntimeFields([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields)] this Type type)
```

Comment:
(nit) Since these annotations are so long, what do you think about making these multiple lines?

Reply:
I don't remember - did we discuss adding shortcut flags to the enum - e.g. instead of having to say `DynamicallyAccessedMemberTypes.PublicFields | DynamicallyAccessedMemberTypes.NonPublicFields` here, I would just say `DynamicallyAccessedMemberTypes.Fields`?

Reply:
I don't think it was discussed at the API review, but it seems like a good idea if this is going to be a common case.